Revise and redirect OAuth expiry guidance (#8654)
fishtown-analytics/docs.getdbt.com: website/docs/faqs/Troubleshooting/refresh-snowflake-oauth-credentials.md

---
title: Reconnecting to Snowflake OAuth after authentication expires
description: "The `authentication has expired` error happens because Snowflake requires tools to renew their OAuth connection at least every 90 days."
sidebar_label: 'Reconnecting to Snowflake OAuth after authentication expires'
---

> When you use [OAuth](/docs/cloud/manage-access/set-up-snowflake-oauth) to connect Snowflake to the dbt platform, <Constant name="cloud" /> stores an refresh token so that your development credentials are usable from tools like the <Constant name="cloud_ide" /> and the dbt Semantic Layer.

Reviewer: ```suggestion
When you connect Snowflake to <Constant name="dbt_platform"/> using [OAuth](/docs/cloud/manage-access/set-up-snowflake-oauth), dbt stores a refresh token. This allows your development credentials to remain usable in tools like the <Constant name="cloud_ide" /> and the dbt Semantic Layer without needing to re-authenticate each time.
```

> If you see a `authentication has expired` error when you try to run queries, you must renew your connection between Snowflake and the dbt platform.

Reviewer: ```suggestion
If you see an `authentication has expired` error when you try to run queries, you must renew your connection between Snowflake and the <Constant name="dbt_platform"/>.
```

To resolve the issue, complete the following steps:

1. Go to your **Profile settings** page, accessible from the navigation menu.

> 2. Navigate to **Credentials** and click on the project you're experiencing the issue with.

Reviewer: ```suggestion
2. Navigate to **Credentials** and then choose the project where you're experiencing the issue.
```

> 3. Under **Development credentials**, click the **Reconnect Snowflake Account** button. This steps you through reauthentication using the SSO workflow.

Reviewer: ```suggestion
3. Under **Development credentials**, click the **Reconnect Snowflake Account** button. This will guide you through re-authenticating using your SSO workflow.
```
